package.json

```diff
@@ -110,7 +110,7 @@
     "yarn-deduplicate": "^3.1.0"
   },
   "engines": {
-    "node": ">=10.13.0"
+    "node": ">=16.13.0"
```

Comment:
but node@14 will be maintained till april 2023

Reply:
hmm... but it's not the LTS anymore